### PR TITLE
Ess gui 5.0.2 release

### PR DESCRIPTION
**Pull request by wpotrzebowski**
PR for 5.0.2 release candidate

### CODE REVIEW

**Comment by wpotrzebowski** [comment]
I've also updated https://github.com/SasView/sasview/wiki/DevNotes_Processeses_DeploymentProcess with instructions how to obtain zenodo id.

File: docs/sphinx-docs/source/user/RELEASE.rst [hunk 1]
```diff
@@ -10,6 +10,40 @@ Release Notes
...
+* Fixes sasview # 1419 Could we improve stepping through graph windows in 5.x
+* Fixes sasview # 1476 Quick Plot in Data Explorer not working for 1D data
+* Fixes sasview # 1463 Dmax explorer : pressing enter closes window
+* Fixes sasview # 1479 linspace errors
+* Fixes sasview # 1444 ESS GUI: Warn user when data set fully masked
+* Fixes sasview # 1459 dy = 1 for every point of Freeze theory curves (and saved as data file) in 5.0.
+* Fixes sasview # 1460 resolution setting not persistent in theory mode of 5.0.1
+* Fixes sasview # 1454 5.0.1 sending same data to more than one fit page - plotting issues
+* Fixes sasview # 1481 Fix for Fit Panel based Plot command for single-data/multiple fit tab
+* Fixes sasmodels # 201 Reparameterize existing model. Refs SasView/sasview#501
```
Comment (krzywon) [inline docs/sphinx-docs/source/user/RELEASE.rst:43]: I'd argue the reparameterization is a new feature, not a bug fix. Also, this was sasmodels PR # 211, not # 201, (I looked it up) and the PR referred to sasview #501.

File: docs/sphinx-docs/source/user/RELEASE.rst [hunk 2]
```diff
@@ -10,6 +10,40 @@ Release Notes
...
+^^^^^^^^^
+* Fixes sasview # 1455: 5.0.1 load project generates a second FitPage1
+* Fixes sasview # 1499: Ess gui 1355 file converter
+* Fixes sasview # 1501: Update 5.x documentation for Corfunc
+* Fixes sasview # 1431 When loading plugin model with a spurious unicode character plugin editor crashes
+* Fixes sasview # 1138 Check package versions in yaml files and setup.py
+* Fixes sasview # 697 Update check_packages to flag required package versions
+* Fixes sasview # 1482 4.x: Check packages comparison to YAML files
+* Fixes sasview # 1484 SasView 5 (Ubuntu) - Need to change colors in Data operation combobox
+* Fixes sasview # 1419 Could we improve stepping through graph windows in 5.x
```
Comment (krzywon) [inline docs/sphinx-docs/source/user/RELEASE.rst:34]: Another line that could potentially be moved to features and improvements, but I'm on the fence with this one.

File: build_tools/generate_zenodo_record.py [hunk 3]
```diff
@@ -5,16 +5,16 @@
 #Replace with live server and live server key
 #DO NOT STORE KEY ON GITHUB
 #TEST settings for Sandbox
-zenodo_url = "https://sandbox.zenodo.org"
-zenodo_api_key = "rdNmQU4ogrtYQKp5jd7238DnigkPu51aMKHa8yFttYtp3GgsxOW8MCfL8Exp"
+zenodo_url = "https://zenodo.org"
+zenodo_api_key = ""
```
[the remark this answers is not in the document]
Comment (rozyczko) [reply build_tools/generate_zenodo_record.py:9]: I think the `sandbox` part makes it a test key only?

[the remark this answers is not in the document]
Comment (wpotrzebowski) [reply build_tools/generate_zenodo_record.py:9]: It is for sandbox. I've removed it from the code so it is more obvious something is missing. How to pass production key is another issue.

File: docs/sphinx-docs/source/user/RELEASE.rst (hunk 1 above)
Comment (wpotrzebowski) [reply docs/sphinx-docs/source/user/RELEASE.rst:43]: Thanks for catching this. It was my original intention to put reparametrization in the new features.

File: build_tools/generate_zenodo_record.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (krzywon) [reply build_tools/generate_zenodo_record.py:9]: I'm just glad it isn't live.

One option for productions keys is to pass them as an argument when running the code. I do this for the saspubs auto-updater. You could store it in a secure file only accessible to the user and code, or, for infrequently run code, just pass it as a string.

[the remark this answers is not in the document]
Comment (wpotrzebowski) [reply build_tools/generate_zenodo_record.py:9]: Good suggestion. I've created an issue #1517 to address for the future release

**Comment by wpotrzebowski** [comment]
Merging it now. We can always improve release notes later.

**Pull request merged**